src/main/java/com/urswolfer/gerrit/client/rest/http/changes/ChangeApiRestClient.java

```diff
@@ -278,9 +284,15 @@ public ChangeInfo get(EnumSet<ListChangesOption> options) throws RestApiExceptio
 
     @Override
     public ChangeInfo get() throws RestApiException {
-        EnumSet<ListChangesOption> options = EnumSet.allOf(ListChangesOption.class);
-        options.remove(ListChangesOption.CHECK);
-        return get(options);
+        String version = getVersion();
+        Set<ListChangesOption> options = EnumSet.allOf(ListChangesOption.class)
```

Comment:
Probably extract this logic into a method in `ListChangesOptionByVersion`?

Reply:
Sure, I'll be more readable.